src/selectors/categorySelector.js

```diff
@@ -0,0 +1,10 @@
+import { createSelector } from 'reselect';
+
+/**
+ * @param {*} state - The initial state from the redux store
+ */
+const categorySelector = state => state.articlesCategoryReducer.category;
```

Comment:
```suggestion
const categorySelector = state => state.categories;
```
Then in the combineReducer, we can have,
`{
  categories: articleCategoryReducer
}`

What do you think?